packages/core/src/extensions/SuggestionMenu/SuggestionPlugin.ts

```diff
@@ -248,6 +248,27 @@ export class SuggestionMenuProseMirrorPlugin<
       },
 
       props: {
+        //STEP 6: the final thing is to listen to the simulated keydown event generated by the onClick of emoji option in slash menu
```

Comment:
Update after reviewing further: I suppose this is for the `onItemClick` slash menu handler below. It's an interesting hack (cool that you got it working), but I think it's a little bit too hacky. imo we should create an API that hides / shows specific suggestionmenus so that we can make this a lot more "idiomatic".

This would also solve some related requests on Discord (https://discord.com/channels/928190961455087667/1204207715929296958). @matthewlipski maybe you could help with this?